RecycleBinThemes.ps1

```diff
@@ -3,9 +3,9 @@
 #
 
 $version = "v1.0.0"
-$pictures_path = [Environment]::GetFolderPath("MyPictures")
+#$pictures_path = [Environment]::GetFolderPath("MyPictures")
```

Comment:
Better to remove this than commenting it out

Reply:
I think it's resolved now